content/tutorials/get-started/get-started-using-php.md

```diff
@@ -0,0 +1,288 @@
+---
+html: get-started-using-php.html
+parent: php.html
+funnel: Build
+doc_type: Tutorials
+category: Get Started
+blurb: Build a PHP app that interacts with the XRP Ledger.
+cta_text: Build an XRP Ledger-connected app
+top_nav_name: PHP
+top_nav_grouping: Get Started
+labels:
+  - Development
+showcase_icon: assets/img/logos/java.svg
+---
+# Get Started Using PHP
+
+This tutorial walks you through the basics of building an XRP Ledger-connected application using [`XRPL_PHP`](https://github.com/AlexanderBuzz/xrpl-php), a PHP library built to interact with the XRP Ledger.
+
+This tutorial is intended for beginners and should take no longer than 30 minutes to complete.
+
+## Learning Goals
+
+In this tutorial, you'll learn:
+
+* The basic building blocks of XRP Ledger-based applications.
+* How to connect to the XRP Ledger using `XRPL_PHP`.
+* How to get an account on the [Testnet](xrp-testnet-faucet.html) using `XRPL_PHP`.
+* How to use the `XRPL_PHP` library to look up information about an account on the XRP Ledger.
+* How to put these steps together to create a simple application.
+
+## Requirements
+
+* `XRPL_PHP` requires PHP 8.1.
+* Recommended for increased performance are the PHP extensions [GMP](http://php.net/manual/en/book.gmp.php) and / or [BCMATH](https://www.php.net/manual/de/book.bc.php)
+
+
+## Installation
+
+`XRPL_PHP` can be installed via [Composer](http://getcomposer.org/doc/00-intro.html):
+
+```console
+composer require hardcastle/xrpl_php
+```
+
+
+## Start Building
+{% set n = cycler(* range(1,99)) %}
+
+When you're working with the XRP Ledger, there are a few things you'll need to manage, whether you're adding XRP to your [account](accounts.html), integrating with the [decentralized exchange](decentralized-exchange.html), or [issuing tokens](tokens.html). This tutorial walks you through basic patterns common to getting started with all of these use cases and provides sample code for implementing them.
+
+Here are the basic steps you'll need to cover for almost any XRP Ledger project:
+
+1. [Connect to the XRP Ledger.](#1-connect-to-the-xrp-ledger)
+1. [Get an account.](#2-get-account)
+1. [Query the XRP Ledger.](#3-query-the-xrp-ledger)
+
+
+### {{n.next()}}. Connect to the XRP Ledger
+
+To make queries and submit transactions, you need to connect to the XRP Ledger. To do this with `XRPL_PHP`, you can use the [`JsonRpcClient`](hhttps://alexanderbuzz.github.io/xrpl-php-docs/client.html):
+
+```php
+<?php
+
+// Use the Composer autoloader
+require __DIR__ . '/vendor/autoload.php';
+
+// Imports 
+use XRPL_PHP\Client\JsonRpcClient;
+
+// Create a Client using the Testnet
+$client = new JsonRpcClient("https://s.altnet.rippletest.net:51234");
+```
```

Comment:
I didn't know until today that it had become standard to omit the `?>` closing tag from PHP scripts. Then again, I've barely touched PHP for the last 10 years.